### PR TITLE
minor changes: start unicorn, start mysql, install meemcached and sta…

### PR DESCRIPTION
**Pull request by vtsetsulin**
…rt it, create directories before copying files into them etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 This is a fork of thoughtbot's laptop script.  It's been modified so it sets up breport.  To use:
 
 1. Install xcode via the app store
+2. Install xcode command line tools using 'xcode-select --install'
 2. Run sh mac script
```
Comment (richardleebr) [inline README.md]: Not a big deal because md figures it out, but make this 3.

File: projects/breport.sh [hunk 2]
```diff
@@ -1,3 +1,5 @@
+APP_ROOT=~/src/breport
```
Comment (richardleebr) [inline projects/breport.sh]: :+1: for refactoring

Comment (tongueroo) [inline projects/breport.sh]: :+1: for cleaning this up :) . looks good, merge whenever please

**Comment by richardleebr** [comment]
This might be a good opportunity to get bflow set up on your computer (you'll need it for future development).  It is a little more process than needed for this change, so totally up to you.

**Comment by tongueroo** [comment]
@richardleebr sounds good, anyone can add!

**Pull request merged**